notifications.py

```diff
@@ -0,0 +1,16 @@
+def median_list(nums):
+    nums = sorted(nums)
+    if len(nums) % 2 == 1:
+        return nums[len(nums)//2]
+    else:
+        return int((nums[len(nums)//2 + 1] + nums[len(nums)//2 - 1])/2.0)
+
+def activity_notification(arr , d):
+    notification_count = 0
+    for i in range(d, len(arr)):
+        median = median_list(arr[i-d:i])
+        if arr[i] >= 2* median:

```

Comment:
There should be a space before '*' to be consistent.

Reply:
Agreed : if arr[i] >= 2 * median: